ibis/mapd/tests/test_client.py

```diff
@@ -84,3 +84,39 @@ def test_union_op(alltypes):
     expr = t1.union(t2)
     with pytest.raises(com.UnsupportedOperationError):
         expr.compile()
+
+
+def test_create_table_schema(con):
+    t_name = 'mytable'
+
+    try:
+        # drop the table if it exists
+        con.drop_table(t_name)
```

Comment:
Does mapd support the `force` flag? This would likely compile to something like `DROP TABLE foo IF EXISTS`. Then you wouldn't need to catch an exception.

Reply:
I check now .. mapd supports that. I am going to change that! thanks!